Jenkinsfile

```
@@ -39,9 +39,12 @@ node ('!(ci-agent-4)') {
           govuk.dockerTagMasterBranch("ckan", env.BRANCH_NAME, env.BUILD_NUMBER)
         }
       }
-
-      stage('Deploy to Integration') {
-        govuk.deployIntegration('ckan', BRANCH_NAME, 'release_' + BUILD_NUMBER, 'deploy')
+    }
+    else {
+      if (env.BRANCH_NAME == 'main-2.9') {
```

This can be simplified to one line:

```suggestion
    else if (env.BRANCH_NAME == 'main-2.9') {
```

---

Thanks, wasn't sure if this was possible, will try that out